JapaneseTokenizer/datamodels.py

```diff
@@ -43,17 +43,19 @@ def convert_list_object(self):
 
 
     class TokenizedResult(object):
-        def __init__(self, analyzed_line, tuple_pos, word_stem, word_surface, is_feature=True, is_surface=False):
-            assert isinstance(analyzed_line, str)
+        def __init__(self, analyzed_line, tuple_pos, word_stem, word_surface, is_feature=True, is_surface=False, misc_info=None):
+            assert isinstance(analyzed_line, (str, type(None)))
```

Comment:
BTW, I think parseToNode is works fine for python3 https://github.com/taku910/mecab/pull/24 , but I don't know whether mecab-python3 supports it.